modules/google-analytics/classes/wp-google-analytics-universal.php

```diff
@@ -49,6 +49,13 @@ public function wp_head() {
 			return;
 		}
 
+		if ( class_exists( 'Jetpack_AMP_Support' )
+			&& Jetpack_AMP_Support::is_amp_request()
+		) {
+			add_filter( 'amp_analytics_entries', 'Jetpack_Google_Analytics::amp_analytics_entries' );
```

Comment:
This filter should be added with a low priority, so that we can check if analytics is already set up using another plugin

Reply:
I agree. I also updated the priority in the legacy class.